src/modules/netinstall/NetInstallPage.cpp

```diff
@@ -62,4 +63,21 @@ void
 NetInstallPage::onActivate()
 {
     ui->groupswidget->setFocus();
+
+    // The NetInstallSelect global sotrage value can be used to make additional items selected by default
+    Calamares::GlobalStorage* gs = Calamares::JobQueue::instance()->globalStorage();
+    if ( gs->contains( "NetinstallSelect" ) && gs->value( "NetinstallSelect" ).canConvert( QVariant::StringList ) )
+    {
+        const QStringList selectNames = gs->value( "NetinstallSelect" ).toStringList();
+
+        static_cast< PackageModel* >( ui->groupswidget->model() )->setSelections( selectNames );
```

Comment:
This is weird. There should be "better" pointers to the model, that don't need casting. `m_config` is there in the `NetInstallPage` class for this reason.